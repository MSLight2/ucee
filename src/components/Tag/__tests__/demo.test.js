import { demoTest } from '../../../tests/shared/demoTest';
import TagDemo from '../demo';

demoTest('Tag', TagDemo);