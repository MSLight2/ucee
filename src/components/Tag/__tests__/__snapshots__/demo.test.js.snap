// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag demo test 1`] = `
Array [
  <span
    className="ucee-tag ucee-tag--primary"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--danger"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--warning"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--success"
    style={Object {}}
  >
    标签
  </span>,
  <div>
    plain
  </div>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--plain"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--danger ucee-tag--plain"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--warning ucee-tag--plain"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--success ucee-tag--plain"
    style={Object {}}
  >
    标签
  </span>,
  <div>
    size
  </div>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--medium"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--large"
    style={Object {}}
  >
    标签
  </span>,
  <div>
    round
  </div>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--round"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--medium ucee-tag--round"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--large ucee-tag--round"
    style={Object {}}
  >
    标签
  </span>,
  <div>
    mark
  </div>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--mark"
    style={Object {}}
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--mark ucee-tag--mark-dir"
    style={Object {}}
  >
    标签
  </span>,
  <div>
    close
  </div>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--close"
    style={Object {}}
  >
    标签
    <i
      className="ucee-icon ucee-icon-close"
      onClick={[Function]}
      style={
        Object {
          "color": "#fff",
          "fontSize": "12px",
        }
      }
    />
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--close"
    style={Object {}}
  >
    标签
    <i
      className="ucee-icon ucee-icon-close"
      onClick={[Function]}
      style={
        Object {
          "color": "#fff",
          "fontSize": "12px",
        }
      }
    />
  </span>,
  <div>
    color
  </div>,
  <span
    className="ucee-tag ucee-tag--primary"
    style={
      Object {
        "backgroundColor": "#19fad7",
      }
    }
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary"
    style={
      Object {
        "backgroundColor": "#d78bde",
        "color": "#e6160b",
      }
    }
  >
    标签
  </span>,
  <span
    className="ucee-tag ucee-tag--primary ucee-tag--plain"
    style={
      Object {
        "backgroundColor": "white",
        "color": "#19fad7",
      }
    }
  >
    标签
  </span>,
]
`;
